### DISCUSSION TITLE
Request: 1 line clarification in the installation instructions

### DISCUSSION THREAD

**Discussion by BetweenTheKeyboardAndChair** [evilsocket/opensnitch#421] [General]
This is a document update request.  I have not learned how to create/fork repositories or create pull requests on github yet.

Please add one line on page:  https://github.com/evilsocket/opensnitch/wiki/Installation

...

tl;dr: use the command line

<< Get the packages from here: <https://github.com/evilsocket/opensnitch/releases>

DEB

    $ sudo dpkg -i opensnitch*.deb python3-opensnitch-ui*.deb; sudo apt -f install

...
--------------------
Without this line, I foolishly spent time looking for opensnitch in any ubuntu/debian repository (fail), and then trying to build it from scratch.  The build of opensnitch failed - on an lrelease bug that appears related to python3-grpcio, and then after uninstalling python3-grpcio, my attempt to pip3 install grpcio==1.16.1 failed with a compilation error.  Installing via your package and command worked beautifully.

I'm on Ubuntu 20.4, Cinnamon, Linux ucin 5.4.0-72-generic.

Thank you for maintaining this app.

**Comment by gustavo-iniguez-goya** [comment]
oops, sorry for that. Ok, I'll add it, thank you!

**Reply by gustavo-iniguez-goya** [reply]
:heavy_check_mark: done

**Reply by BetweenTheKeyboardAndChair** [reply]
Thank you.
